CHANGES.txt

```diff
@@ -9,6 +9,11 @@ Unreleased
    If you are using crate with Python 2.7 or 3.3 already, you will not be able
    to install newer versions of this package.
 
+ - BREAKING: In the testing layer, the custom setting of
```

Comment:
The initial change was to add the flood_stage watermark setting, which was a breaking changes. However, in the end, we decided to remove the other settings, which is not a breaking change.

Reply:
I get what you mean. Still, if you rely on these default settings, it is breaking and not backwards-compatible.